src/controllers/selectDirector.controller.ts

```diff
@@ -91,19 +91,30 @@ export class SelectDirectorController extends BaseController {
   }
 
   private updateSession(body: SelectDirectorFormModel, selectedDirector?: Optional<DirectorDetails>): void {
+    const session: DissolutionSession = this.session.getDissolutionSession(this.httpContext.request)!
+
+    if (!this.hasFormChanged(body, session)) {
```

Comment:
This has been agreed with Jayti. If the user has not changed anything on the page, then do not rewrite to session.